cliboa/template/Pipfile.above310

```diff
@@ -40,6 +40,7 @@ python-gnupg = "==0.4.8"
 openpyxl = "==3.0.9"
 pyminizip = "==0.2.5"
 psycopg2 = "==2.9.1"
+numpy = "<2"
```

Comment:
I also made some changes to this p-r to pass the unit test.  
If there are any conflicts, I will deal with them.